flow/scripts/sc/tools/openroad/sc_apr.tcl

```diff
@@ -27,13 +27,17 @@ set inputs [list]
 # Step-specific pre-processing step(s)
 #if {$sc_step == "or_synth_hier_report"} {
 if {$sc_step == "or_synth"} {
-    # Pre-synthesis: mark dont-use cells in liberty files, and merge them.
+    # Pre-synthesis: decompress liberty files, and merge them.
     foreach f [split $::env(LIB_FILES)] {
-        exec $::env(UTILS_DIR)/markDontUse.py -p $::env(DONT_USE_CELLS) -i $f -o "../../[file tail $f]-mod.lib"
+        if {[file extension $f] == .gz} {
+            exec gunzip -c $f > "../../[file tail $f]-mod.lib"
+        } else {
+            file link -symbolic $f "../../[file tail $f]-mod.lib"
+        }
```

Comment:
@gadfort can you review this please? I don't know how to test.

Reply:
@maliberty We went down a rabbithole og symlinks and tcl. TL;DR symlinks cant be made to work w makefiles and tcl file timestamps are truncated to whole seconds.

Does that cause a proble here?

Reply:
This is linking the .lib file which should usually come directly from the manufacturer/vendor. Generally it's not expected to be changed by the user, maybe only during a `git pull` (and rarely at that). If it does matter, it can just be a `cp` command instead.

Reply:
the sym link is fine here.
Although I'm not sure if any of this code is actually functional at this point

Reply:
Okay, will keep it as is. Thanks.